source/screens/Home/Transactions/TransactionListItem.tsx

```diff
@@ -35,11 +43,25 @@ const TransactionListItem = (props: Props) => {
 
   return (
     <li className="transaction-list-item">
-      <span>{type.slice(0, 1).toUpperCase() + type.slice(1)}</span>
-      <span className="transaction-list-item__balance-impact">
+      <div className="row">
+        <span>{type.slice(0, 1).toUpperCase() + type.slice(1)} </span>
+
+        {!blockHash && (
+          <p className="transaction-list-item-unconfirmed">Unconfirmed</p>
+        )}
+      </div>
+      <span
+        className={classnames('transaction-list-item__balance-impact', {
+          'transaction-list-item__balance-impact--negative': isNegative(
```

Comment:
For negativity check please use math over `(satoshisBalanceImpact - feeImpact)` instead of string manipulation